package/system/uci/patches/001-fix-compile-error-with-lua5.3.patch

```diff
@@ -0,0 +1,15 @@
+--- a/lua/uci.c
++++ b/lua/uci.c
+@@ -34,10 +34,10 @@
+ #define DPRINTF(...) do {} while (0)
+ #endif
+ 
+-#if !defined LUA_VERSION_NUM || LUA_VERSION_NUM==501
+-
+ int luaopen_uci(lua_State *L);
+ 
++#if !defined LUA_VERSION_NUM || LUA_VERSION_NUM==501
```

Comment:
Please create a separate pull request against the uci repository or send it over the mailing list to integrate this into libuci. 
https://github.com/openwrt/uci/